docs/troubleshoot/troubleshoot-apiml-error-codes.md

```diff
@@ -981,15 +1029,15 @@ The following error message codes may appear on logs or API responses. Use the f
 
 ### ZWEAG108E
 
-  z/OSMF instance '%s' not found or incorrectly configured.
+  z/OSMF instance '%s' not found or incorrectly configured. Gateway is shutting down.
 
   **Reason:**
 
-  The Gateway could not find the z/OSMF instance from the Discovery Service.
+  The Gateway could not find the z/OSMF instance from the Discovery Service or it could not communicate with provided z/OSMF instance.
 
   **Action:**
 
-  Ensure that the z/OSMF instance is configured correctly and that it is successfully registered to the Discovery Service.
+  Ensure that the z/OSMF instance is configured correctly and that it is successfully registered to the Discovery Service and that API Mediation Layer can communicate with provided z/OSMF instance.
```

Comment:
Add 'the': "...and that API Mediation Layer can communicate with **the** provided z/OSMF instance."